### PR TITLE
Issue #KN-517 feat: Video player web component integration in knowlg app and portal

### PR DESCRIPTION
**Pull request by santoshilimi**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,6 +12,9 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/reflect-metadata/0.1.13/Reflect.min.js"
   integrity="sha512-jvbPH2TH5BSZumEfOJZn9IV+5bSwwN+qG4dvthYe3KCGC3/9HmxZ4phADbt9Pfcp+XSyyfc2vGZ/RMsSUZ9tbQ=="
   crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"
```
Comment (HarishGangula) [inline src/index.html]: Can use this from NPM

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (santoshilimi) [reply src/index.html]: Yes, updated

**Review by HarishGangula — APPROVED** [review]

File: knowlg-app/package.json [hunk 2]
```diff
@@ -38,11 +35,9 @@
     "ngx-bootstrap": "8.0.0",
     "npm": "^8.19.1",
     "rxjs": "~6.6.0",
-    "shared": "file:../shared-0.0.1.tgz",
+    "shared": "file:../shared-0.0.2.tgz",
     "tslib": "^2.2.0",
     "video.js": "^7.20.2",
```
Comment (vinukumar-vs) [inline knowlg-app/package.json]: Why video.js dependency is added here?

File: knowlg-app/src/app/player/player.page.ts [hunk 3]
```diff
@@ -27,7 +27,7 @@ export class PlayerPage implements OnInit {
     });
   }
   playerEvent(event) {
-    const eventType = _.get(event, 'edata.type');
+    const eventType = _.get(event, 'detail.edata.type');
```
Comment (vinukumar-vs) [inline knowlg-app/src/app/player/player.page.ts:30]: Why is this detail is added now?

Comment (santoshilimi) [reply knowlg-app/src/app/player/player.page.ts:30]: As we implemented Web component the event details comes as part of **detail** , FYI 
Screenshot 2022-11-18 at 11 24 02 AM

File: knowlg-app/package.json (hunk 2 above)
Comment (santoshilimi) [reply knowlg-app/package.json]: Removed video.js dependency

**Review by HarishGangula — CHANGES_REQUESTED** [review]
When you are integrating the video player component all the dependencies should be in single JS file

**Review by vinukumar-vs — APPROVED** [review]

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**